sycl/test/basic_tests/reduction_ctor.cpp

```diff
@@ -49,14 +49,24 @@ void testKnown(T Identity, BinaryOperation BOp, T A, T B) {
   Q.submit([&](handler &CGH) {
     // Reduction needs a global_buffer accessor as a parameter.
     // This accessor is not really used in this test.
+    accessor<T, Dim, access::mode::read_write, access::target::global_buffer>
+        ReduRWAcc(ReduBuf, CGH);
     accessor<T, Dim, access::mode::discard_write, access::target::global_buffer>
-        ReduAcc(ReduBuf, CGH);
-    auto Redu = ONEAPI::reduction(ReduAcc, BOp);
-    assert(toBool(Redu.getIdentity() == Identity) &&
+        ReduDWAcc(ReduBuf, CGH);
+    auto ReduRW = ONEAPI::reduction(ReduRWAcc, BOp);
+    auto ReduDW = ONEAPI::reduction(ReduDWAcc, BOp);
+    auto ReduRW2020 = sycl::reduction(ReduBuf, CGH, BOp);
```

Comment:
Should there be a test for `sycl::reduction(USM Pointer, BOp)` as well?

Reply:
Right. Fixed in 0cdf646139cb6f27a28f985cd123eb96ebd540be